worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageLockWrapper.java

```diff
@@ -43,7 +38,13 @@ public JdbcLockRegistry contractLockRegistry(LockRepository lockRepository) {
 
     public LockRepository contractLockRepository() {
         final DefaultLockRepository defaultLockRepository = new DefaultLockRepository(dataSource);
-        defaultLockRepository.setTimeToLive(LOCK_TIME);
+
+        // A lock is automatically killed, regardless of whether it is in use
+        // if it has not been renewed TTL seconds after it was created.
+        // What this means is that if you are locking longer than this TTL, then
+        // you need to renew the lock otherwise you will lose it and get undefined
+        // behavior when you attempt to unlock your lock.
+        defaultLockRepository.setTimeToLive(10 * TEN_MINUTES);
```

Comment:
Why 10 * ten minutes?

Why not just have a different constant specified to be more clear as to the intent?